src/containerapp/azext_containerapp/_help.py

```diff
@@ -500,14 +500,23 @@
     short-summary: Commands to manage certificates for the Container Apps environment.
 """
 
+helps['containerapp env certificate create'] = """
+    type: command
+    short-summary: Create a managed certificate.
+    examples:
+    - name: Create a managed certificate.
+      text: |
+          az containerapp env certificate create -g MyResourceGroup --name MyEnvironment --certificate-name MyCertificate --hostname MyHostname --validation-method CNAME
+"""
+
 helps['containerapp env certificate list'] = """
     type: command
     short-summary: List certificates for an environment.
     examples:
     - name: List certificates for an environment.
       text: |
           az containerapp env certificate list -g MyResourceGroup --name MyEnvironment
-    - name: List certificates by certificate id.
+    - name: List certificate by certificate id.
```

Comment:
Why use the singular nouns?

Reply:
If certificate id is provided, there will be only one certificate returned.

Reply:
OK, but in fact, this operation will be more appropriate in the show command instead of list command

Reply:
I see. We will consider having a show command for certificate. Thanks!